.travis.yml

```diff
@@ -8,3 +8,9 @@ install: mvn -e install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 
 jdk:
   - oraclejdk8
+  
+deploy:
+  provider: bintray
+  file: descriptor.json
+  user: vaani
+  key: 315213b83b21bc121e7ff476148a598193db810b
```

Comment:
Again[[1]](http://stackoverflow.com/questions/32356238/issue-using-encrypted-bintray-key-in-travis-yml) this looks like our password is in the open.